index.js

```diff
@@ -53,6 +82,18 @@ module.exports = {
             if (prefix === '_BuffEx_') {
                 return new Buffer(JSON.parse(value.slice(8)));
             }
+            if (prefix === '_IuFrRa_') {
+                var value = JSON.parse(value.slice(8));
+                var newInstance = null;
+                (() => {
+                    var _class = eval(`${value.constructor}`);
+                    var _inst = parse(value.instance);
+
+                    newInstance = new _class();
```

Comment:
So I was not sure what I wanted to do here when I was thinking about it before. There may be something along the lines of what is found in Reflect.construct but on the other hand, we already have the property layout of the constructed instance. 

Given we do not know what parameters were passed in upon the instance's original creation we may just want to create an object from the set of properties we found and use Reflect.setPrototypeOf on the appropriate bits to achieve a situation where `newInstance` is a `value.instance` that responds true to `newInstance instanceof [_class]`; if that makes any sense at all.